### PR TITLE
Fixed #29734 -- Added option to sort message strings for translations by msgid.

### PR DESCRIPTION
**Pull request by kosmos342**
Ticket [#29734](https://code.djangoproject.com/ticket/29734)

### CODE REVIEW

**Comment by timgraham** [comment]
Thanks for the contribution. A [Trac ticket](https://code.djangoproject.com/) and a test are required.  See also our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).

File: docs/ref/django-admin.txt [hunk 1]
```diff
@@ -699,6 +699,11 @@ is:
 
 Requires ``gettext`` 0.19 or newer.
 
+.. django-admin-option:: --sort-output
```
Comment (charettes) [inline docs/ref/django-admin.txt]: That'll need a `.. versionadded:: 2.2`.

Comment (kosmos342) [reply docs/ref/django-admin.txt]: updated and rebased

**Comment by Joxos** [comment]
Yes,it’s good.

 

发件人: noreply@github.com <noreply@github.com> 代表 Carlton Gibson
发送时间: 2018年9月6日 16:35
收件人: django/django <django@noreply.github.com>
抄送: Subscribed <subscribed@noreply.github.com>
主题: Re: [django/django] Fixed #29734 -- Added option to sort message strings for translations by msgid. (#9295)

 

@carltongibson approved this pull request.

OK, looks good.

**Comment by timgraham** [comment]
Closing per ticket.

**Pull request closed without merging**